detox/test/e2e/27.retries.js

```diff
@@ -0,0 +1,17 @@
+jest.retryTimes(3);
+
+describe('jest.retryTimes() support', () => {
+  let counter = 3;
+
+  beforeAll(async () => {
+    await device.launchApp({ newInstance: true });
+  });
+
+  it('should fail twice and pass once', async () => {
+    const matcher = --counter > 0
+      ? by.text('Not existing')
+      : by.text('Sanity');
+
+    await element(matcher).tap();
+  });
+});
```

Comment:
Nice, but here too (much like with screenshot tests) we lack the testing of the final artifacts...

Reply:
Also, I didn't see anything related in the build's artifacts. Have I been looking in the wrong place?

Reply:
Hmmm... well, I can add checks, theoretically.

Reply:
So action items here:
1. Rename to `27.retries.test.js`
2. Add artifacts validation?

Reply:
Thanks for pointing out, fixing.